torchgeo/datasets/advance.py

```diff
@@ -117,7 +117,7 @@ def __init__(
             )
 
         self.files = self._load_files(self.root)
-        self.classes = sorted(set(f["cls"] for f in self.files))
+        self.classes = sorted({f["cls"] for f in self.files})
```

Comment:
Not a fan of this change, `set(...)` is a nice explicit way of doing things!

Reply:
I think I actually prefer set comprehension on this one. Anyway, it's just personal preference. I actually prefer single quotes over double quotes but black gets what black wants lol.